config/spotbugs/exclude.xml

```diff
@@ -18,6 +18,17 @@
     for the mapping from SpotBugs rank to verbal labels.
 -->
 <FindBugsFilter>
+    <Match>
+        <!-- MongoDB status: "No Fix Needed", SpotBugs rank: * -->
```

Comment:
@stIncMale how did you determine the ranks?  Above comment says to look at https://spotbugs.readthedocs.io/en/latest/filter.html#rank but I don't see anything useful at that location.

Reply:
I determined the ranks as described in https://github.com/mongodb/mongo-java-driver/pull/1392:

> To get the rank data, I had enabled SpotBugs XML reports, then was commenting out parts of exclude.xml and manually get the rank data from the XML reports.

Reply:
> Above comment says to look at https://spotbugs.readthedocs.io/en/latest/filter.html#rank but I don't see anything useful at that location.

It says to look at the page "for the mapping from SpotBugs rank to verbal labels", and the mapping is there. This is in case we decide to map the numbers to something more human-friendly.

Reply:
Ah, got it. I added the ranks and a comment saying how to find them, for our future selves.